docs-source/content/samples/simple/domains/model-in-image/initial.md

```diff
@@ -262,6 +262,7 @@ If you don't see the `weblogic-deploy.zip` file, then you missed a step in the [
 Now, you use the Image Tool to create an image named `model-in-image:WLS-v1` that's layered on a base WebLogic image. You've already set up this tool during the prerequisite steps.
 
 Run the following commands to create the model image and verify that it worked:
+Note: If you are creating a WebLogic model-in-image, add --chown oracle:root to imagetool.sh command. When creating FMS images, do not use the --chown option.
```

Comment:
When creating FMW Infrastructure images,

Reply:
Fixed it.